### PR TITLE
S3 Assets query example

### PR DESCRIPTION
**Pull request by jcharcalla**
Added and example for retrieving S3 bucket assets costs.

### CODE REVIEW

File: assets-api.md [hunk 1]
```diff
@@ -410,3 +410,93 @@ Retrieve assets cost data for the past seven days, aggregated by type, and as cu
...
+{
+  "code": 200,
+  "data": [
+    {
+       "AWS/297945954695/__undefined__/Storage/__undefined__/Cloud/AmazonS3/cloud-bench-1/__undefined__": {
+        "type": "Cloud",
+        "properties": {
+          "category": "Storage",
+          "provider": "AWS",
+          "account": "297945954695",
```
Comment (srpomeroy) [inline assets-api.md:432]: I suggest we obscure the account IDs in the response.

**Comment by bstuder99** [comment]
Aside from Sean's comment, LGTM

File: assets-api.md (hunk 1 above)
Comment (jcharcalla) [reply assets-api.md]: Good call! I have removed them.

**Review by srpomeroy — APPROVED** [review]
Lgtm

**Pull request merged**